code/src/components/Form.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+
+import WelcomePage from 'components/WelcomePage'
+import RadioButton from 'components/RadioButton'
+import CheckBox from 'components/CheckBox';
+import DropDown from 'components/DropDown';
+import Summary from 'components/Summary';
+
+export const Form = () => {
+// STATES // 
+  const [ageGroup, setAgeGroup] = useState('')
+  const [personality, setPersonality] = useState(false)
+  const [person, setPerson] = useState(false)
+  const [actor, setActor] = useState('')
+  const [name, setName] = useState('')
+  const [counter, setCounter] = useState(0)
+
+// DECLARED FUNCTIONS //
+  const handleNameChange = (event) => {
+        setName(event.target.value) 
+      }
+
+  const handleRadioChange = (event) => { 
+        setAgeGroup(event.target.value)
+      }
+
+  const handlePersonality = (event) => {
+        setPersonality(event.target.checked)
+      }
+  
+  const handlePerson = (event) => {
+        setPerson(event.target.checked)
+      }
+
+  const handleDropDown = (event) => {
+        setActor(event.target.value) 
+      }
+
+  const handleNextQuestion = () => {
+        setCounter(counter + 1)
+      }
+
+  return (
+    <section className='form-wrapper'>
+      {counter === 0 &&
+        <WelcomePage
+          nextQuestion={handleNextQuestion}
+          onNameChange={handleNameChange}
+          username={name}
+      />}
```

Comment:
Nicely solved with the multistep survey with conditional rendering of the pages!!